code/__HELPERS/game.dm

```diff
@@ -26,6 +26,11 @@
 		return null
 	return format_text ? format_text(A.name) : A.name
 
+/proc/get_coordinates_string(var/atom/A)
+	var/turf/T = get_turf(A)
+	return A.loc ? "[T.x],[T.y],[T.z]" : "nullspace"
```

Comment:
```suggestion
	return T ? "[T.x],[T.y],[T.z]" : "nullspace"
```
Just because A is inside a loc doesn't mean that loc isn't in nullspace. (example: dying inside a nullspaced mech)

Reply:
Oh yeah, good point.